flink-scala/src/main/java/org/apache/flink/api/scala/typeutils/Tuple2CaseClassSerializerSnapshot.java

```diff
@@ -48,7 +48,7 @@
 
 	@SuppressWarnings("unused")
 	public Tuple2CaseClassSerializerSnapshot() {
-		super(correspondingSerializerClass());
+		super(package$.MODULE$.tuple2ClassForJava());
```

Comment:
I don't want to nitpick here, but I think it was more readable they way it was before.

Reply:
Makes sense. For this class I'll keep it as is.